[DTM] Added the default value implementation for property. (#1391)
SoftwareAG/c8y-docs: content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md

If the type is "enumeration", then the input field must be in the list of values specified during creation of the asset property.
If the type is "text" or "number" and custom criteria were provided during asset property creation, then the input value in CSV template must fulfill all the asset property criteria.

> A property can be added with a default value during its creation. If a property has an empty value in CSV file then it will display default value after asset creation. To override the display of default value in created asset then pass null as value in CSV file, this will display undefined as property value after asset creation.

Reviewer: When creating a property, you have the option to assign a default value. If a property's value is left empty in the CSV file during bulk import, its value will be set to default. To override this behavior, enter 'null' as the value in the CSV file. This doesn't set the value and this will be seen as undefined when viewing the asset.

> When creating a property, you have the option to assign a default value. If a property's value is left empty in the CSV file during bulk import, its value will be set to default. To override this behavior, enter 'null' as the value in the CSV file. This doesn't set the value and this will be seen as undefined when viewing the asset.

Reviewer 2: ```suggestion
When creating a property, you have the option to assign a default value. If a property's value is left empty in the CSV file during the bulk import, its value is set to default. To override this behavior, enter 'null' as the value in the CSV file. This does not set the value and it is seen as undefined when viewing the asset.
```

Reviewer 2: @spau6 What is meant with "null", do you meant zero (0)?

Author: It is the string null
